### PR TITLE
Fix/side param

### PR DESCRIPTION
**Pull request by PetervDooren**
merge together with https://github.com/tue-robotics/tue_robocup/pull/1083

### CODE REVIEW

File: custom/skills.yaml [hunk 1]
```diff
@@ -56,19 +56,13 @@ skills:
...
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-        right_suction:
-            grasp_offset:  {x: -0.0338, y: 0.012, z: 0.0844, roll: 0, pitch: 0.87239795, yaw: -1.57079632679}
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
+        base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
+
+    gripper:
+        grasp_frame: 'grippoint_left'
```
Comment (MatthijsBurgh) [inline custom/skills.yaml]: `arm_center` and `grippoint_left` are conflicting

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: custom/skills.yaml (hunk 1 above)
Comment (PetervDooren) [reply custom/skills.yaml]: Agreed, will see if I can change the name of this frame

**Review by MatthijsBurgh — DISMISSED** [review]

File: custom/skills.yaml [hunk 2]
```diff
@@ -59,6 +59,7 @@ skills:
         base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
 
     gripper:
+        ac_gripper: 'hero/gripper/action'
```
Comment (MatthijsBurgh) [inline custom/skills.yaml]: Add leading slash

Comment (PetervDooren) [reply custom/skills.yaml]:
```suggestion
        ac_gripper: '/hero/gripper/action'
```

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**